app/api/actions/post.actions.ts

```diff
@@ -101,10 +101,13 @@ export const UpdateMedia = async (id: string, data: FormData) => {
   });
 };
 
-export const DeletePost = async (id: string) => {
+export const DeletePost = async (id: string, userId: string) => {
   await httpRequest<void>(`/posts/${id}`, {
     method: 'DELETE',
   });
+
+  revalidatePath('/');
+  revalidatePath(`/posts/${userId}`);
```

Comment:
I am not sure if this is right. Pages under post have the postId as a url param and not the user id. Other than that, please specify the type of the revalidation path (eg. "page")

Reply:
fixed